enterprise/backend/enterprise_core/views.py

```diff
@@ -203,8 +203,11 @@ def get_build(request):
     LICENSE_SEATS = settings.LICENSE_SEATS
     LICENSE_EXPIRATION = settings.LICENSE_EXPIRATION
     try:
-        expiration_iso = datetime.fromisoformat(LICENSE_EXPIRATION)
-        license_expiration = date_format(expiration_iso, use_l10n=True)
+        if LICENSE_EXPIRATION and LICENSE_EXPIRATION.lower() != "unset":
```

Comment:
I would advise to wrap the date cast in a try/except block, instead of explicitly checking for an "unset" string.